Sources/Rendering/Misc/FullScreenRenderWindow/index.js

```diff
@@ -72,17 +71,17 @@ function vtkFullScreenRenderWindow(publicAPI, model) {
   model.renderer = vtkRenderer.newInstance();
   model.renderWindow.addRenderer(model.renderer);
 
-  // OpenGlRenderWindow
-  model.openGLRenderWindow = vtkOpenGLRenderWindow.newInstance();
-  model.openGLRenderWindow.setContainer(model.container);
-  model.renderWindow.addView(model.openGLRenderWindow);
+  // apiSpecificRenderWindow
+  model.apiSpecificRenderWindow = model.renderWindow.newAPISpecificView();
```

Comment:
We could use the url parameter extractor to select the implementation. That way, we can run the same example but based on the URL pick one implementation vs the other.